src/Utils.ts

```diff
@@ -32,6 +53,46 @@ export function boolString(bool: boolean): string {
   return bool ? chalk.green('good'.padEnd(14)) : chalk.red('wrong'.padEnd(14));
 }
 
+export async function getPastEventsForHub(
+  parameters: ManagerEventParameters,
+  { fromBlock, toBlock }: PastEventOptions,
+  names: ManagerEvent[] = DEFAULT_MANAGER_EVENTS
+): Promise<Array<TypedEvent>> {
+  const provider = getProvider();
+
+  const relyHubAddress = config.get<string>('contracts.relayHubAddress');
```

Comment:
I would prefer to store the config keys into constants to be reused elsewhere

Reply:
updated